libs/application/template-api-modules/src/lib/modules/template-api.module.ts

```diff
@@ -1,18 +1,24 @@
 import { DynamicModule } from '@nestjs/common'
 
 // Imports of custom template API modules
-import { modules } from './templates'
+import { modules, clientModules, clientConfigs } from './templates'
 
 import { BaseTemplateAPIModuleConfig } from '../types'
 
 import { TemplateAPIService } from './template-api.service'
+import { ConfigModule } from '@island.is/nest/config'
 
 export class TemplateAPIModule {
   static register(config: BaseTemplateAPIModuleConfig): DynamicModule {
     return {
       module: TemplateAPIModule,
       imports: [
+        ...Object.values(clientModules).map((Module) => Module),
         ...Object.values(modules).map((Module) => Module.register(config)),
+        ConfigModule.forRoot({
+          isGlobal: true,
+          load: [...Object.values(clientConfigs).map((Config) => Config)],
```

Comment:
Same here.

Reply:
Maybe we should import ConfigModule further up in ApplicationSystemApi, especially since it's `forRoot` with `isGlobal: true`.